app/contexts/location-and-service/LocationAndService.context.tsx

```diff
@@ -12,26 +13,30 @@ type LocationType = {
     state: string;
 };
 
-type LocationAndServiceProviderState = {
+type LocationAndServiceContextType = {
     location: LocationType;
     locationTextInput: string;
-    service: string;
     setLocation: (location: LocationType) => void;
     setLocationTextInput: (locationTextInput: string) => void;
+    service: string;
     setService: (service: string) => void;
 };
 
-const initialState: LocationAndServiceProviderState = {
+const initialState: LocationAndServiceContextType = {
```

Comment:
Let's also keep this name as it was since we're calling this `const initialState` here. And since we can already see that the object is a type, we don't need to specify twice. Calling it `LocationAndServiceProviderState` is more clear that we're housing our state variables here

```suggestion
const initialState: LocationAndServiceContextState = {
```

Reply:
Ok, I thought it'd be more clear and consistent with how I see we've been defining types for stuff?
As in `Thing: TypeofThing`, or am I understanding it wrong?

Reply:
Well I guess it depends on the context (no pun intended :P). But in this case since it's in the context, I guess it makes more sense to me to name it state since it has all our state. Type would be used more for times like when you need it to define a type (excuse my bad explanation lmao), but basically like how you used `BusinessType` in the `BusinessGrid` file

Reply:
I'm not sure I totally understand it, but I'll take your word for it!

Reply:
![Screen Shot 2024-06-21 at 6 00 15 PM](https://github.com/cindyquach12/beautyhub/assets/106289033/6dadcd5e-8b0a-4323-9a55-478477cda7c2)
Another example is like this, we name it props b/c we're passing in those values as props to the function, even though it is also a type. This is an example of what I mean when I say it depends on the context (how you're using it)

Reply:
![Screen Shot 2024-06-21 at 6 04 39 PM](https://github.com/cindyquach12/beautyhub/assets/106289033/9e2b9519-5076-4922-87d0-303a5e28481d)
![Screen Shot 2024-06-21 at 6 04 57 PM](https://github.com/cindyquach12/beautyhub/assets/106289033/2aea17f1-55c9-4208-b25c-f00bd82d2581)
The way you're using it here is strictly for the type. In this case, it wouldn't make much sense to pass in `BusinessType` to the `FunctionComponent` as props right? Like in that example above ^
I'm talking about the naming. If you were to pass BusinessType in as props to the FC then you'd change the name to `BusinessGridProps`

Reply:
Thanks for the explanation!